qt/aqt/data/qt/icons/BUILD.bazel

```diff
@@ -56,9 +57,32 @@ copy_mdi_icons(
     ],
 )
 
+py_binary(
+    name = "color_svg",
+    srcs = [
+        "color_svg.py",
+        "//qt/aqt:colors",
+    ],
+    imports = ["."],
+    visibility = [":__subpackages__"],
+)
+
+color_svg(
+    name = "chevron-down",
+)
+
+color_svg(
+    name = "drag-horizontal",
+    color = "FG_SUBTLE",
+)
+
 filegroup(
     name = "icons",
-    srcs = ["mdi-icons"] + glob([
+    srcs = [
+        "mdi-icons",
+        "chevron-down",
+        "drag-horizontal",
```

Comment:
This is currently including base.svg as well as base-light.svg and base-dark.svg. We could avoid that by creating a separate mdi-icons that holds the files we plan to modify, and then not including it in this filegroup.